main/src/com/google/refine/model/Recon.java

```diff
@@ -51,8 +51,14 @@
  
 public class Recon implements HasFields, Jsonizable {
     
+    /**
+     * Freebase schema URLs kept for compatibility with legacy reconciliation results
+     */
     private static final String FREEBASE_SCHEMA_SPACE = "http://rdf.freebase.com/ns/type.object.id";
```

Comment:
freebase still available?

Reply:
No, but it can still be referred to: Freebase ids are still in use, for instance on Wikidata. There are still a number of places in the code (and possibly in other extensions / reconciliation services) where this is used. Are you proposing to delete this, and if so why?

Reply:
The Freebase namespace is still referenced out in the wild.  keep this.